package.json

```diff
@@ -36,10 +36,14 @@
     "eslint": "^6.3.0",
     "eslint-config-prettier": "^6.3.0",
     "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jest": "^22.17.0",
```

Comment:
sorry to be picky, but the new devDependencies and the rename of the index.js file, don't belong to this commit